### ISSUE TITLE
bug:  string is identified as a column if enable planer_v2

### ISSUE THREAD

**Issue by sandflee** [datafuselabs/databend#7057]
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

a65c8d332

### What's Wrong?

string is identified as a column,   if I disable planer_v2 or change "RUSSIA" to 'RUSSIA' , everthing is ok;

```
ERROR 1105 (HY000): Code: 1065, displayText = error:
  --> SQL:1:58
  |
1 | select * from hive.default.customer_p2 where c_nation2 = "RUSSIA"
  |                                                           ^^^^^^^^ column doesn't exist
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by sundy-li** [comment]
Now databend uses `single` quote string as literal string.

**Comment by sandflee** [comment]
hive sql use both single and double quote string : (

**Comment by leiysky** [comment]
> is that any plan to make double quote string as literal string as MySQL did?

We had a proposal for SQL dialect #5372, but haven't put it on schedule. 

The double-quoted identifier follows standard SQL specification, and many of the analytical databases are in this style, e.g. Snowflake, AWS Redshift, Azure Synapse and etc.

**Comment by leiysky** [comment]
Closed by #7175
